docs/3.0.0/docs/canton/usermanual/performance.rst

```diff
@@ -298,6 +298,66 @@ as a result, the command processing consumes resources even though some commands
 If you choose lower resource limits, you may observe a lower latency, at the cost of lower throughput and
 commands getting rejected with the error code ``PARTICIPANT_BACKPRESSURE``.
 
+.. _tuning_ledger_api_limits:
+
+**Tune resource limits at the ledger api level.** Resource limits can also be imposed on the ledger api level.
+As these settings are applied closer to the ledger applications, they can be used for protecting the resources
+of individual participants rather than the entire Canton system.
```

Comment:
```suggestion
**Tune resource limits at the ledger API level.** Resource limits can also be imposed on the ledger API level.
As these settings are applied closer to the ledger applications, they can be used for protecting the resources
of individual participants rather than the entire Canton system.
```